internal/app/tfsec/checks/gke_abac_enabled.go

```diff
@@ -0,0 +1,33 @@
+package checks
+
+import (
+	"fmt"
+
+	"github.com/liamg/tfsec/internal/app/tfsec/parser"
+	"github.com/liamg/tfsec/internal/app/tfsec/scanner"
+)
+
+// GkeAbacEnabled See https://github.com/liamg/tfsec#included-checks for check info
+const GkeAbacEnabled scanner.CheckCode = "GKE001"
```

Comment:
I didn't make this clear anywhere so apologies, but the check code should be per cloud provider e.g. google/azure/amazon. I think:
```suggestion
const GkeAbacEnabled scanner.CheckCode = "GCP004"
```